Header header

uint32 MAX_NR_OBJECT = 100
Object[100] objects
